Yoast/javascript
# Add hebrew function words

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Adds Hebrew function words.
* [Yoast SEO Free] Improves keyphrase recognition in Hebrew by filtering the function words such as `אבל`, `כול`, `זה`.

## Relevant technical choices:

- I've added specs for function word recognition in `buildTopicStemsSpec`. I'd suggest to add new function word specs there in the future. Previously, we tested function words via the relevant words specs, see e.g. this PR: https://github.com/Yoast/javascript/pull/691/files Since internal linking doesn't make use of this functionality anymore, I don't think it makes sense to add specs there. I did, however, confirm that nothibng breaks if not all the `filteredAt` categories are exported as we do in other languages by temporarily adding a `relevantWords` spec for Hebrew.
- See the [frequency list](https://docs.google.com/spreadsheets/d/1nstlPwrzShx9L3672R1js-5XB7u45cgnusajJCZXzjM/edit?usp=sharing) to get a sense for how the words added compare with the most frequent words.

## Test instructions

This PR can be tested by following these steps:

Test the following in WP SEO Free:
* Set the site language to Hebrew.
* Create a new post and add a text with at least 150 words.
* Set your keyphrase to `כל החתולים`.
* Add the word `החתולים` to the text.
* Make sure that the keyphrase density shows that the keyphrase is found once.


## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


https://yoast.atlassian.net/browse/LIN-496
